packages/admin/src/components/pageRouting/Pages.tsx

```diff
@@ -79,6 +85,10 @@ export const Pages = ({ children, layout }: PagesProps) => {
 						return Object.entries(v.type.getPages(v.props))
 					} else if (isPageProviderElement(v)) {
 						return [[v.type.getPageName(v.props, pageName), () => v]]
+					} else if (k.endsWith('.tsx') || k.endsWith('.jsx')) {
+						const Lazy = lazy(v)
+						const WithFallback = () => <Suspense fallback={<ContainerSpinner />}><Lazy /></Suspense>
+						return [[k.slice(k.lastIndexOf('/') + 1, -4), WithFallback]]
```

Comment:
I am thinking if it wouldn't be logical to structure the pages like this:
```
pages/
├─ article/
│  ├─ list.tsx
│  ├─ create.tsx
│  └─ edit.tsx
└─ page/
│  ├─ list.tsx
│  ├─ create.tsx
│  └─ edit.tsx
└─ homepage.tsx
```

Vite's glob would support this, but this splitting would create collision between page's and article's edit page.

Reply:
well, i guess we could allow `/` in page name

Reply:
should be working now

Reply:
`pages` could be named `routes` to overcome comfusion between `page` content type and admin `pages` used for routing.